### ISSUE TITLE
failed assert 'new_col >= 0'

### ISSUE THREAD

**Issue by pieterkockx** [neovim/neovim#4916]
- `nvim --version`: 0.1.4
- Vim (version:7.4 ) behaves differently? yes
- Operating system/version:ARCH 4.6
- Terminal name/version: tty
- `$TERM`: linux

### Actual behaviour
failed assert 'new_col >= 0' in ui_cursor_left(), ui.c:461

### Expected behaviour
print empty line

### Steps to reproduce using `nvim -u NORC`

nvim -u NORC
:!echo ^H
_^H is a (backspace) character generated using CTRL-V CTRL-H_

**Comment by oni-link** [comment]
Looks like a duplicate of #4142.

**Comment by mhinz** [comment]
Yup, duplicate. Thanks for reporting anyway.

**Comment by justinmk** [comment]
https://github.com/neovim/neovim/pull/4376 fixes this issue, I should break out the relevant commit to unblock that.
